thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/tools/CleanupAndRegenerateAnomaliesTool.java

```diff
@@ -168,6 +204,10 @@ private void deleteRawResults(List<RawAnomalyResultDTO> rawResults) {
   private void deleteMergedResults(List<MergedAnomalyResultDTO> mergedResults) {
     LOG.info("Deleting merged results");
     for (MergedAnomalyResultDTO mergedResult : mergedResults) {
+      // Delete raw anomalies of the merged anomaly
```

Comment:
The use case I am thinking of is to just replay the merging logic, where you do not need to remove raw anomalies.